go-controller/pkg/ovn/obj_retry.go

```diff
@@ -572,7 +639,12 @@ func (oc *Controller) addResource(objectsToRetry *retryObjs, obj interface{}, fr
 			extraParameters.portGroupIngressDenyName,
 			extraParameters.portGroupEgressDenyName,
 			obj)
-
+	case factory.EgressFirewallType:
+		egressFirewall := obj.(*egressfirewall.EgressFirewall)
+		if err := oc.addEgressFirewall(egressFirewall); err != nil {
+			klog.Errorf("Failed to create egress firewall %s, error: %v", getEgressFirewallNamespacedName(egressFirewall), err)
```

Comment:
no reason to log here, the caller will log the error. Just return this as your error